test/request-response-samples-test.js

```diff
@@ -36,31 +36,6 @@ var requestContext = {
   }
 };
 
-var origRandom = Math.random;
-module.exports = {
```

Comment:
Tried to explain the purpose in my commit message:
> Changed how tests are generated to avoid global before/after hooks. By using the format with exporting before, beforeEach and after hooks aswell as the test suite one actually register the hooks globally - applying them to every test run by npm test. This wasn't really a problem until you try to do setTimeouts, which will never resolve since timekeeper.freeze is used (https://github.com/vesln/timekeeper#featuresproblems)"

Before it was attached to the global before hook, which meant that every test in the entire suite would get a fake random - if they wanted it or not. Now I limited the fake random to just be for the "request-response-sample" tests. The biggest problem was with timekeeper, blowing up any attempt to use setTimeouts in a test.